README.md

```diff
@@ -155,28 +155,43 @@ The ⓒ mark means that [`@uppy/companion`](https://uppy.io/docs/companion), a s
   <img src="https://saucelabs.com/browser-matrix/transloadit-uppy.svg" alt="Sauce Test Status"/>
 </a>
 
-We aim to support IE11 and recent versions of Safari, Edge, Chrome, Firefox and Opera.
+We aim to support recent versions of Safari, Edge, Chrome, Firefox and Opera.
 
-We still run end-to-end tests with IE10, but we are not actively supporting it or fixing visual / minor issues.
+We still run end-to-end tests with IE11, but we are not actively supporting it or fixing visual / minor issues.
 
 ### Polyfills
 
-Uppy heavily uses Promises. If your target environment [does not support Promises](https://caniuse.com/#feat=promises), use a polyfill like `es6-promise` before initializing Uppy.
+Here's a list of polyfills you'll need to include to make Uppy work in older browser (e.g., IE11):
 
-When using remote providers like Google Drive or Dropbox, the Fetch API is used. If your target environment does not support the [Fetch API](https://caniuse.com/#feat=fetch), use a polyfill like `whatwg-fetch` before initializing Uppy. The Fetch API polyfill must be loaded _after_ the Promises polyfill, because Fetch uses Promises.
+- [abortcontroller-polyfill](https://github.com/mo/abortcontroller-polyfill)
+- [es6-promise](https://github.com/stefanpenner/es6-promise)
+- [math-log2](https://github.com/sindresorhus/math-log2)
+- [md-gum-polyfill](https://github.com/mozdevs/mediaDevices-getUserMedia-polyfill)
+- [resize-observer-polyfill](https://github.com/que-etc/resize-observer-polyfill)
+- [url-polyfill](https://github.com/lifaon74/url-polyfill)
+- [whatwg-fetch](https://github.com/github/fetch)
 
-With a module bundler, you can use the required polyfills like so:
+If you're using a bundler, you need import them before Uppy:
 
-```shell
-npm install es6-promise whatwg-fetch
-```
 ```js
-require('es6-promise/auto')
-require('whatwg-fetch')
-const Uppy = require('@uppy/core')
+import 'es6-promise/auto';
```

Comment:
Uppy is not going to move to ESM in `2.0` so probably best to keep these examples with `require`.

Reply:
I disagree, I think we should start to push for ESM actually. ESM can import both ESM and CJS – except in Deno, because they don't support CJS at all – us internally using CJS for the time being shouldn't refrain our users to switch to ESM (we really want that to happen before 3.0 actually).

Reply:
Fair enough! But then I would argue for consistency, so _every_ README and documentation example. Perhaps a bit too much for this PR?

Reply:
Agreed! I've assigned https://github.com/transloadit/uppy/issues/2951 to myself.